cmd/mysterium_client/command_run/command.go

```diff
@@ -71,7 +74,9 @@ func (cmd *CommandRun) Run(options CommandOptions) (err error) {
 		return err
 	}
 
-	apiEndpoints := tequilapi.NewApiEndpoints()
+	idm := identity.NewIdentityManager("TEST")
+	router := tequilapi.NewApiEndpoints()
+	router.GET("/list-identities", endpoints.ListIdentitiesEndpointFactory(idm).ListIdentities)
```

Comment:
Read on RESTful resources naming conventions:
- https://restfulapi.net/resource-naming/
- http://www.restapitutorial.com/lessons/restfulresourcenaming.html

Reply:
changed to '/identities'